waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/TSetIpAddressProcessorFactory.java

```diff
@@ -47,6 +53,9 @@ public TSetIpAddressProcessorFactory(
   @Override
   public TProcessor getProcessor(TTransport transport) {
     try {
+      Socket socket = ((TSocket) transport).getSocket();
+      log.debug("Opened a connection from ip: {}", socket.getInetAddress().getHostAddress());
```

Comment:
Opened a connection from ip -> Received a connection from IP
(I think that reads a bit better, NABD either way)

Reply:
I agree